doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -199,10 +210,110 @@ async def _start_build(self, metadata: ImageMetadata, build_repo: BuildRepo, dyn
         )
         pipelinerun = cast(resource.ResourceInstance, pipelinerun)
 
-        pipelinerun_name = pipelinerun['metadata']['name']
-        self._logger.info(f"[%s] Created PipelineRun: {pipelinerun_name}", metadata.distgit_key)
+        self._logger.info(f"[%s] Created PipelineRun: {self.build_pipeline_url(pipelinerun)}", metadata.distgit_key)
         return pipelinerun
 
+    @staticmethod
+    def build_pipeline_url(pipelinerun):
+        pipelinerun_name = pipelinerun['metadata']['name']
+        application = pipelinerun['metadata']['labels']['appstudio.openshift.io/application']
+        return (f"{constants.KONFLUX_UI_HOST}/application-pipeline/"
+                f"workspaces/{constants.KONFLUX_UI_DEFAULT_WORKSPACE}/"
+                f"applications/{application}/"
+                f"pipelineruns/{pipelinerun_name}")
+
+    def update_konflux_db(self, metadata, build_repo, pipelinerun, outcome):
+        if not metadata.runtime.konflux_db:
+            self._logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux '
+                                 'DB.')
+            return
+
+        try:
+            rebase_repo_url = build_repo.https_url
+            rebase_commit = build_repo.commit_hash
+
+            df_path = build_repo.local_dir.joinpath("Dockerfile")
+            df = DockerfileParser(str(df_path))
+
+            source_repo = df.labels['io.openshift.build.source-location']
+            commitish = df.labels['io.openshift.build.commit.id']
+
+            component_name = df.labels['com.redhat.component']
+            version = df.labels['version']
+            release = df.labels['release']
+            nvr = "-".join([component_name, version, release])
+
+            pipelinerun_name = pipelinerun['metadata']['name']
+            build_pipeline_url = self.build_pipeline_url(pipelinerun)
+
+            build_record_params = {
```

Comment:
I noticed we're not storing `installed_packages` and `parent_images`, which is crucial for the ocp4-scan implementation. Are we planning to add this at a later time?

Reply:
I'm okay with merging this and quickly following up